build-ova.sh

```diff
@@ -123,7 +115,6 @@ function copy_ova() {
 }
 
 function main() {
-    checkout_image_builder_branch
```

Comment:
why is this not required ??

Reply:
We are already syncing the code to the image builder commit ID that we wanted, so we don't need to sync it again when building the image.